Source/WebGPU/WebGPU/BindGroup.mm

```diff
@@ -157,28 +160,37 @@ static MTLRenderStages metalRenderStage(ShaderStage shaderStage)
         [argumentEncoder[stage] setArgumentBuffer:argumentBuffer[stage] offset:0];
     }
 
+    // FIXME: The bind group layout determines the layout of what gets put into the bind group.
```

Comment:
If we iterate over the bind group layout, then I don't think we need the comments on lines 190-193, they are essentially referring to the same issue.

In either case, can we please add a bugzilla link here?